docker-compose-services/postgres/README.md

```diff
@@ -52,6 +52,58 @@ hooks:
 There are also another non-plain-text formats that `pg_dump` can generate, and you might need to work with them. If that's the case, there is also
 a `ddev pg_restore` command that will restore `.ddev/pgsql-db/postgresql.db.dump` into `db`.
 
+## Multiple databases
+
+* By default, DDEV adds a single database called `db`. But it's possible to create additional databases when you first initialize this service.
+
+* The offical way to add multiple databases is outlined on the [docker image](https://hub.docker.com/_/postgres) under the "Initialization scripts"
+
+To implement this in DDEV:
+
+* Add an additional volume to `./.ddev/docker-composer.postgres.yaml`
+
+  ```yaml
+      volumes:
+      ...
+      - "./pgsql-db:/docker-entrypoint-initdb.d"
+  ```
+
+* Create `./.ddev/pgsql-db` folder if it doesn't already exist
+
+  ```bash
+  mkdir ./.ddev/pgsql-db -p
+  ```
+
+* Add your *.sql,*.sql.gz, or *.sh files to `./.ddev/pgsql-db`. EG. Create a file called `./.ddev/pgsql-db/create_testing_database.sql` and add the following:
+
+  ```sql
+  CREATE DATABASE testing
```

Comment:
I think we should pick a name as mentioned before and run with it consistently in the example?